Documentation/Experimental/SceneUnderstanding/README_SceneUnderstanding.md

```diff
@@ -0,0 +1,99 @@
+# MRTK Scene Understanding Demo
+
+This is an __experminental__ prelease that integrates [Scene Understanding](https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding) (SU) as a MRTK.
+
+SU - in a nutshell, will return a semantic representation of scene entities as well as their geometric forms on __HoloLens 2__.
```

Comment:
Maybe elaborate further. As an example, the SU observer can provide simplified planes to represent walls, floors, etc identified in an environment at runtime. Or something to that attestment to bring it down to layman's terms. Also, I would recommending making the beginning

Spatial Understanding will return a semantic....